### PR TITLE
feat(scripts): added merkleproof gatekeeper to signup script and utils

### PR DESCRIPTION
**Pull request by crisgarner**
# Description

Added merkleproof gatekeeper to signup script and utils

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://maci.pse.dev/docs/contributing) and [code of conduct](https://maci.pse.dev/docs/contributing/code-of-conduct).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

File: packages/cli/ts/commands/signup.ts [hunk 1]
```diff
@@ -301,3 +303,26 @@ export const getHatsSingleGatekeeperData = async ({
     hatsContract: hatsContract.toString(),
   };
 };
+
+/**
```
Comment (ctrlc03) [inline packages/cli/ts/commands/signup.ts:307]: typo

**Review by ctrlc03 — COMMENTED** [review]
Thanks just a typo to fix

File: packages/cli/ts/commands/signup.ts [hunk 2]
```diff
@@ -301,3 +303,26 @@ export const getHatsSingleGatekeeperData = async ({
...
+ * Get the zupass gatekeeper data
```
Comment (ctrlc03) [inline packages/cli/ts/commands/signup.ts:308]: change from zupass to the correct one

Comment (crisgarner) [reply packages/cli/ts/commands/signup.ts]: thanks fixed

**Review by ctrlc03 — APPROVED** [review]
Thanks

**Pull request merged**